Formula/apib.rb

```diff
@@ -14,28 +13,21 @@ class Apib < Formula
     sha256 "bbe9bc25a8584f163347662675d78b69cdfaac495be5f2fa026dfca112f8d4a4" => :sierra
   end
 
-  depends_on "apr"
-  depends_on "apr-util"
+  depends_on "cmake" => :build
+  depends_on "libev"
   depends_on "openssl@1.1"
 
   def install
-    # Fix detection of libcrypto for OpenSSL 1.1
-    inreplace "configure", "CRYPTO_num_locks", "EVP_sha1"
+    ENV.append "EXTRA_INCLUDE_DIR", "${prefix}/include"
+    ENV.append "EXTRA_LIB_DIR", "${prefix}/lib"
+    ENV.append "OPENSSL_INCLUDE_DIR", "#{Formula["openssl@1.1"]}/include"
+    ENV.append "OPENSSL_LIB_DIR", "#{Formula["openssl@1.1"]}/lib"
 
-    # Upstream hardcodes finding apr in /usr/include
-    # https://github.com/apigee/apib/issues/11
-    inreplace "configure" do |s|
-      s.gsub! "/usr/include/apr-1.0", "#{Formula["apr"].opt_libexec}/include/apr-1"
-      s.gsub! "/usr/include/apr-1", "#{Formula["apr"].opt_libexec}/include/apr-1"
-    end
-    ENV.append "LDFLAGS", "-L#{Formula["apr-util"].opt_libexec}/lib"
-    ENV.append "LDFLAGS", "-L#{Formula["apr"].opt_libexec}/lib"
-    ENV.append "CFLAGS", "-I#{Formula["apr"].opt_libexec}/include/apr-1"
-    ENV.append "CFLAGS", "-I#{Formula["apr-util"].opt_libexec}/include/apr-1"
+    mkdir "release"
+    system "cmake", ".", "-DCMAKE_BUILD_TYPE=Release"
```

Comment:
Looks good. One change I'd make:

```suggestion
    system "cmake", ".", *std_cmake_args
```